docs/source/markdown/podman-push.1.md.in

```diff
@@ -103,29 +103,32 @@ Add a sigstore signature at the destination using a private key at the specified
 
 ## EXAMPLE
 
-This example pushes the image specified by the imageID to a local directory in docker format.
-
- `# podman push imageID dir:/path/to/image`
-
-This example pushes the image specified by the imageID to a local directory in oci format.
-
- `# podman push imageID oci-archive:/path/to/layout:image:tag`
-
-This example pushes the image specified by the imageID to a container registry named registry.example.com
-
- `# podman push imageID docker://registry.example.com/repository:tag`
-
-This example pushes the image specified by the imageID to a container registry named registry.example.com and saves the digest in the specified digestfile.
+Push specified image to a local directory in docker format:
```

Comment:
```suggestion
Push specified image to a local directory:
```

The format depends on the one that `imageID` has.  I find the _name_ "imageID" a bit strange.